base/system/services/security.c

```diff
@@ -54,6 +54,7 @@ static
 DWORD
 ScmCreateSids(VOID)
 {
+    SID_IDENTIFIER_AUTHORITY WorldAuthority = {SECURITY_WORLD_SID_AUTHORITY};
     SID_IDENTIFIER_AUTHORITY NullAuthority = {SECURITY_NULL_SID_AUTHORITY};
```

Comment:
```suggestion
    SID_IDENTIFIER_AUTHORITY NullAuthority = {SECURITY_NULL_SID_AUTHORITY};
    SID_IDENTIFIER_AUTHORITY WorldAuthority = {SECURITY_WORLD_SID_AUTHORITY};
```
(sorted by increased SID values, see e.g. https://git.reactos.org/?p=reactos.git;a=blob;f=base/services/svchost/globals.c;hb=2f2795ab4cf7fdddf6c9d88d885a338a7678a888#l22 or the headers declarations)